digital-deck-ui/src/components/CardDeckImageEnum.js

```diff
@@ -1,9 +1,66 @@
 export default {
-    FULL: 'https://i.imgur.com/Ggu8xlE.png',
-    HALF: 'https://i.imgur.com/4pqATmt.png',
-    FIVE: 'https://i.imgur.com/fP9bL6O.png',
-    FOUR: 'https://i.imgur.com/tPJcCL9.png',
-    THREE: 'https://i.imgur.com/83dtG4X.png',
-    TWO: 'https://i.imgur.com/zIQRKqa.png',
-    ONE: 'https://i.imgur.com/TZJdpPC.png'
-};
\ No newline at end of file
+    BACK: {
```

Comment:
I originally intended for this file to just be assets for the card deck. I think you should either put the assets for the face of the cards in a different file or rename this file to something like `CardImageEnum.js` to be more accurate.

Reply:
I think the name still implies to whole deck, be it front or back

Reply:
I guess it's fine. The name just doesn't really click with me